doc/source/whatsnew/v0.25.0.rst

```diff
@@ -405,6 +405,7 @@ Reshaping
 - Bug in :func:`merge` where merging with equivalent Categorical dtypes was raising an error (:issue:`22501`)
 - Bug in :class:`DataFrame` constructor when passing non-empty tuples would cause a segmentation fault (:issue:`25691`)
 - Bug in :func:`pandas.cut` where large bins could incorrectly raise an error due to an integer overflow (:issue:`26045`)
+- Bug in :func:`DataFrame.sort_index` where sorting with the first index level last was throwing an error (:issue:`26053`)
```

Comment:
this is not giving me any idea what you are changing, can you reword a bit

Reply:
> Bug in :func:`DataFrame.sort_index` where an error is thrown when a multi-indexed 
> DataFrame is sorted on all levels with the initial level sorted last (:issue:`26053`)
is this one better?

Reply:
sure